### ISSUE TITLE
Duda compress

### ISSUE THREAD

**Issue by Nicolas413** [IIC2133-PUC/2023-2#101]
Hola tengo una duda en compress, se debe buscar el alpha tal las hojas del quadtree sean menores pero a la vez lo mas proximo al h que nos entregan?

**Comment by CarloGauss33** [comment]
Claro. Al final como se estan tratando de redondear el alpha y no todos los valores son doubles. Seria el int más cercano (Encontrado con busqueda binaria)
